src/test/java/teammates/test/cases/action/FeedbackParticipantFeedbackResponseCommentDeleteActionTest.java

```diff
@@ -0,0 +1,263 @@
+                // non-existent feedback response comment id
+                Const.ParamsNames.FEEDBACK_RESPONSE_COMMENT_ID, "123123123123123",
+        };
+
+        action = getAction(submissionParams);
+        result = getAjaxResult(action);
+
+        data = (FeedbackResponseCommentAjaxPageData) result.data;
+
+        assertFalse(data.isError);
+        assertNull(feedbackResponseCommentsDb.getFeedbackResponseComment(feedbackResponseComment.feedbackResponseId,
+                feedbackResponseComment.commentGiver, feedbackResponseComment.createdAt));
+        assertEquals("", result.getStatusMessage());
+
+        ______TS("Typical successful case when feedback participant is a student");
+
+        questionNumber = 3;
+        feedbackQuestion = feedbackQuestionsDb.getFeedbackQuestion(
+                "First Session", "idOfCourse1", questionNumber);
+        giverEmail = "student1InCourse1@gmail.tmt";
+        receiverEmail = "student1InCourse1@gmail.tmt";
+        feedbackResponse = feedbackResponsesDb.getFeedbackResponse(feedbackQuestion.getId(), giverEmail, receiverEmail);
+
+        feedbackResponseComment = dataBundle.feedbackResponseComments.get("comment1FromStudent1");
+
+        feedbackResponseComment = feedbackResponseCommentsDb.getFeedbackResponseComment(feedbackResponse.getId(),
+                feedbackResponseComment.commentGiver, feedbackResponseComment.createdAt);
+        assertNotNull("response comment not found", feedbackResponseComment);
+
+        StudentAttributes student = dataBundle.students.get("student1InCourse1");
+        gaeSimulation.loginAsStudent(student.googleId);
+
+        submissionParams = new String[] {
+                Const.ParamsNames.COURSE_ID, feedbackResponseComment.courseId,
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, feedbackResponseComment.feedbackSessionName,
+                Const.ParamsNames.FEEDBACK_RESPONSE_ID, feedbackResponseComment.feedbackResponseId,
+                Const.ParamsNames.FEEDBACK_RESPONSE_COMMENT_ID, feedbackResponseComment.getId().toString(),
+                Const.ParamsNames.FEEDBACK_RESPONSE_COMMENT_TEXT, feedbackResponseComment.commentText + " (Edited)",
+        };
+        result = getAjaxResult(getAction(submissionParams));
+        data = (FeedbackResponseCommentAjaxPageData) result.data;
+
+        assertFalse(data.isError);
+        assertNull(feedbackResponseCommentsDb.getFeedbackResponseComment(feedbackResponseComment.feedbackResponseId,
+                feedbackResponseComment.commentGiver, feedbackResponseComment.createdAt));
+        assertEquals("", result.getStatusMessage());
+
+    }
+
+    @Override
+    protected FeedbackParticipantFeedbackResponseCommentDeleteAction getAction(String... params) {
+        return (FeedbackParticipantFeedbackResponseCommentDeleteAction) gaeSimulation.getActionObject(getActionUri(),
+                params);
+    }
+
+    @Override
+    @Test
+    protected void testAccessControl() throws Exception {
```

Comment:
similar here, better to have separate unit tests and use this one to call them.

Reply:
`testExecuteAndPostProcess()` is testing that. Isn't that sufficient?